src/ngx_http_lua_util.c

```diff
@@ -2008,8 +2008,45 @@ ngx_http_lua_escape_uri(u_char *dst, u_char *src, size_t size, ngx_uint_t type)
 
                     /* mail_auth is the same as memcached */
 
+    static uint32_t header_name[] = {
```

Comment:
Style: needs 2 spaces before the identifier `header_name`. See above for examples (like `memcached[]`).